PARAMETERS = (
    # TOPICS
    ("camera_topic", "/nra/camera"),
    ("cones_topic", "/nra/cones"),
    ("path_topic", "/nra/path"),
    ("imu_topic", "/nra/imu"),
    ("lidar_topic", "/nra/lidar"),
    ("depth_topic", "/nra/depth"),
    ("control_topic", "/nra/control"),
    ("odom_topic", "/nra/odom"),
    ("track_topic", "/nra/track"),
    ("reset_topic", "/nra/reset"),
    # CONFIGURATION
    ("time_step", 5.0),
)
